pynamodb/models.py

```diff
@@ -542,16 +542,25 @@ def count(cls,
             non_key_attribute_classes=non_key_attribute_classes,
             filters=filters)
 
-        data = cls._get_connection().query(
-            hash_key,
-            index_name=index_name,
-            consistent_read=consistent_read,
-            key_conditions=key_conditions,
-            query_filters=query_filters,
-            limit=limit,
-            select=COUNT
-        )
-        return data.get(CAMEL_COUNT)
+        count_buffer = 0
+        last_evaluated_key = None
+        started = False
+        while not started or last_evaluated_key:
```

Comment:
can't you save the `started` var if you just set `last_evaluated_key` to a sentinel value?